sklearn/model_selection/_split.py

```diff
@@ -1481,6 +1481,27 @@ class GroupShuffleSplit(ShuffleSplit):
         If None, the random number generator is the RandomState instance used
         by `np.random`.
 
+    Examples
+    --------
+    >>> import numpy as np
+    >>> from sklearn.model_selection import GroupShuffleSplit
+    >>> X = np.array([[1, 2], [3, 4], [5, 6]])
+    >>> y = np.array([1, 2, 1])
+    >>> groups = np.array([1, 2, 3])
```

Comment:
Should we showcase adjust this example to showcase what happens when there is more than one sample in a group? For example:

```py
import numpy as np
from sklearn.model_selection import GroupShuffleSplit
X = np.ones(shape=(8, 2))
y = np.array([0, 1, 0, 1, 0, 1, 0, 1])
groups = np.array([1, 1, 1, 2, 2, 3, 3, 3])
gss = GroupShuffleSplit(n_splits=2, train_size=.6, random_state=42)
for train_idx, test_idx in gss.split(X, y, groups):
    print("TRAIN:", train_idx, "TEST:", test_idx)
```

Reply:
+1 also showing `train_idx`, and `test_idx` is enough I think, the actual content of `X`, `y` doesn't matter for this example.

Reply:
I honestly just had a look at how other model selection examples are presented. 

`LeavePGroupsOut` was my main source in this case.

Reply:
@thomasjpfan / @rth Gotta say this is a more insightful example. Should I check if I can adjust `LeavePGroupsOut`?